packages/babel-plugin-ember-test-metadata/__tests__/utils-test.js

```diff
@@ -32,111 +30,112 @@ describe('getNodeProperty', () => {
 });
 
 describe('getNormalizedFilePath', () => {
-  const fileOpts = {
-    embroiderBuildPath: {
-      root: '',
-      filename: path.join(
+  const appName = "example-app";
+  const moduleName = "example-app";
+  const addonName = "example-addon";
+  const addonPath = path.join('lib', addonName);
+  const config = {
+    pkg: {
+      name: appName
+    },
+  }
+
+  describe("given file path from a classic build", () => {
```

Comment:
I don't think you need the describes, but just a single test for each of the scenarios. If we'd rather have separate utils for each of the normalizers, that's fine too.